core/domain/state_domain.py

```diff
@@ -503,8 +503,10 @@ def get_all_html_content_strings(self):
         if self.id in (
                 'ItemSelectionInput', 'MultipleChoiceInput',
                 'DragAndDropSortInput'):
-            customization_args_html_list = (
+            customization_args_translatable_html_list = (
                 self.customization_args['choices']['value'])
+            customization_args_html_list = [translatable_html['html'] for (
```

Comment:
Ditto, here and elsewhere. Though I'm happy to chat more about what might make a good convention -- it's possible that translatable_html is the best we can do.

Reply:
Sure, I've added this to our meeting minutes.